<launch>

  <arg name="UAV_NAME" default="$(env UAV_NAME)"/>
  <arg name="custom_config" default="" />

  <group ns="$(arg UAV_NAME)">

    <node name="path_random_flier" pkg="nodelet" type="nodelet" args="standalone mrs_uav_testing/PathRandomFlier" output="screen" >

      <rosparam file="$(find mrs_uav_testing)/config/path_random_flier.yaml" />

      <!-- Load a user param file -->
      <rosparam if="$(eval not arg('custom_config') == '')" file="$(arg custom_config)" />

      <param name="uav_name" value="$(arg UAV_NAME)" />

        <!-- Subscribers -->
      <remap from="~position_command_in" to="control_manager/position_cmd" />
      <remap from="~control_manager_diag_in" to="control_manager/diagnostics" />
      <remap from="~mpc_prediction_in" to="control_manager/mpc_tracker/prediction_full_state" />

        <!-- Services -->
      <remap from="~activate_in" to="automatic_start/start" />
      <remap from="~path_out" to="trajectory_generation/path" />

    </node>

  </group>
</launch>
